helm/examples/metrics/README.md

```diff
@@ -187,37 +187,41 @@ sink will report metrics from all components to the same index. Therefore, the
 index must be configured to receive metrics from all components. 
 
 #### Index Configuration
-The index must be created in ElasticSearch before metrics can be reported to it. The name is passed
-to the sink as a configuration setting. The index must be created with the following settings:
+The sink requires the index to be created and configured fully in ElasticSearch in order for the sink to report
```

Comment:
The README updates are meant to be helpful, but I found the wording confusing. It might be worth seeking feedback from the doc guys.